sys-apps/fakeroot-ng/files/fakeroot-ng-0.18-remove-ptrace-request.patch

```diff
@@ -0,0 +1,38 @@
+__ptrace_request is defined as an enum in glibc. This replaces
+__ptrace_request with int to make it portable.
```

Comment:
Good explanation, but let's link to the musl ML posts I showed too.

Reply:
Done!